app/models/course/assessment/answer.rb

```diff
@@ -2,7 +2,7 @@
 class Course::Assessment::Answer < ActiveRecord::Base
   include Workflow
   actable
-  acts_as :discussion_topic, class_name: Course::Discussion::Topic.name
+  acts_as_discussion_topic display_global: true
```

Comment:
@allenwq @lowjoel - Maybe we can keep it simple here and just use `global` - so we have the below. I prefer keeping the API's short, and explaining it in the comments. What do you all think?

```ruby
acts_as_discussion_topic global: true

scope :global, (lambda do
...
)
```